js/src/modals/Verification/email-store.js

```diff
@@ -54,7 +54,7 @@ export default class EmailVerificationStore extends VerificationStore {
   }
 
   constructor (api, account, isTestnet) {
-    super(api, EmailVerificationABI, 'emailverification3', account, isTestnet);
+    super(api, EmailVerificationABI, 4, account, isTestnet);
```

Comment:
fully agree, that's why i had changed it to string in the first place. constants is a good solution i think.